Frida/Script.swift

```diff
@@ -214,4 +251,55 @@ public class Script: NSObject, NSCopying {
     private let releaseConnection: GClosureNotify = { data, _ in
         Unmanaged<SignalConnection<Script>>.fromOpaque(data!).release()
     }
+    
+    // MARK: - RPC Types
+    
```

Comment:
Trailing whitespace. (Xcode is sadly very good at inserting those.)

Reply:
Yeah, it truly is. Maybe I'll find a script to clean my commits next time.